CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.4] - 2023-06-09
```

Comment:
So your changes just go under the "Unreleased" section.  The new version heading is created automatically during the release process, and items moved automatically.